src/compute/runtime/src/blueprint.ts

```diff
@@ -0,0 +1,79 @@
+import logger from "./logger";
+
+const methods = ['get', 'post', 'patch', 'del', 'options', 'put']
+
+interface Bp {
+    [key: string]: Array<{
+        httpMethod: string,
+        constructor: any,
+        handler: string
+    }>
+}
+interface BP {
```

Comment:
Can we give `Bp` and `BP` more meaningful names? They're quite confusing at the moment.

Reply:
It becomes the definition of routing, the meaning is clearer